blocks/story-carousel-block/_index.scss

```diff
@@ -0,0 +1,25 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-story-carousel {
+	&__main-title {
+		@include scss.block-properties("category-carousel-title");
+		@include scss.block-components("category-carousel-title");
```

Comment:
```suggestion
		@include scss.block-properties("story-carousel-title");
		@include scss.block-components("story-carousel-title");
```

Reply:
Haha you know what I was doing now ;-). Done